npm2deb/scripts.py

```diff
@@ -320,5 +325,20 @@ def _show_mapper_warnings():
         print("")
 
 
+def send_itp(args):
+    mail_itp(str(args.node_module), str(get_npm2deb_instance(args).get_ITP()))
+
+
+def mail_itp(node_module=None, template=None):
+    try:
+        send_to = 'submit@bugs.debian.org'
+        sub = '\"ITP: ' + str(node_module) + ' -- FIX_ME write the Debian package description\"'
+        mail_body = '\"'+'\n'.join(template.split('\n')[3:])+'\"'
+        open_mail = 'xdg-email --body '+str(mail_body)+' --subject '+str(sub)+' '+str(send_to)
```

Comment:
Not sure that `xdg-email` is a good choice: it is not installed by default _(recommended by task-desktop, installed by chromium or gimp)_, then this will fail in a text-only install

Reply:
Is it sent automatically by default ?

Reply:
> Not sure that `xdg-email` is a good choice: it is not installed by default _(recommended by task-desktop, installed by chromium or gimp)_, then this will fail in a text-only install

I guess it comes with xdg-utils, but if this isn't a good choice, what would you suggest as a alternative?

Reply:
> Is it sent automatically by default ?

No, we will need to run ```npm2deb senditpmail <node-module-name>``` to send the ITP

Reply:
> > Not sure that `xdg-email` is a good choice: it is not installed by default _(recommended by task-desktop, installed by chromium or gimp)_, then this will fail in a text-only install
> 
> I guess it comes with xdg-utils, but if this isn't a good choice, what would you suggest as a alternative?

Yes but xdg-utils isn't installed by default. Could you use simply "editor + mail", always available _(like reportbug behavior)_ ?

Reply:
> > > Not sure that `xdg-email` is a good choice: it is not installed by default _(recommended by task-desktop, installed by chromium or gimp)_, then this will fail in a text-only install
> > 
> > 
> > I guess it comes with xdg-utils, but if this isn't a good choice, what would you suggest as a alternative?
> 
> Yes but xdg-utils isn't installed by default. Could you use simply "editor + mail", always available _(like reportbug behavior)_ ?

Yes. But if I use mail, the user will have to edit the ITP file locally first, and then it would be mailed. I wanted to add the feature in a way that there's no ITP file downloaded,(trying npm2deb to not depend on local files) rather the ITP data is fetched and the mail is opened in a e-mail client like thuderbird, wherein user can directly edit and send the mail. Is there a way to go about it? Or should I discard the idea and go about with mail _along with_ editor?